extensions/github/package.nls.json

```diff
@@ -1,7 +1,7 @@
 {
 	"displayName": "GitHub",
 	"description": "GitHub features for VS Code",
-	"config.gitAuthentication": "Controls whether to enable automatic GitHub authentication for git commands within VS Code.",
+	"config.gitAuthentication": "Controls whether to enable automatic GitHub authentication for git commands within Azure Data Studio.",
```

Comment:
I don't think it's possible to add a carbon edit tag to json file, do we want to update this one as well?

Reply:
This is fine - the tags are mostly just for information during merges anyways.